### PR TITLE
feat(clerk-js): Retheme org general page layout

### PR DESCRIPTION
**Pull request by desiprisg**
## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

File: packages/clerk-js/src/ui/constants.ts [hunk 1]
```diff
@@ -4,6 +4,6 @@ export const USER_PROFILE_NAVBAR_ROUTE_ID = {
 };
 
 export const ORGANIZATION_PROFILE_NAVBAR_ROUTE_ID = {
+  GENERAL: 'general',
```
Comment (anagstef) [inline packages/clerk-js/src/ui/constants.ts:7]: This is a breaking change. But I think we are ok to introduce this change.

File: package-lock.json
Comment (anagstef) [inline package-lock.json:1]: @desiprisg We should not commit the package-lock

File: packages/clerk-js/src/ui/components/OrganizationProfile/AddDomainForm.tsx [hunk 2]
```diff
@@ -34,34 +45,51 @@ export const AddDomainPage = withCardStateProvider(() => {
...
+      .then(async res => {
+        setDomainId(res.id);
+        domains?.revalidate?.();
         if (res.verification && res.verification.status === 'verified') {
-          return navigate(`../domain/${res.id}?mode=select`);
+          onSuccess();
+        } else {
+          wizard.nextStep();
         }
-        return navigate(`../domain/${res.id}/verify`);
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/OrganizationProfile/AddDomainForm.tsx:41]: @desiprisg are we good to be replacing these?

File: packages/clerk-js/src/ui/components/OrganizationProfile/AddDomainForm.tsx [hunk 3]
```diff
@@ -34,34 +45,51 @@ export const AddDomainPage = withCardStateProvider(() => {
...
       })
       .catch(err => {
+        console.log(err);
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/OrganizationProfile/AddDomainForm.tsx:58]: Is this a leftover or we want to log here?

File: packages/clerk-js/src/ui/components/OrganizationProfile/VerifyDomainForm.tsx [hunk 4]
```diff
@@ -59,8 +64,11 @@ export const VerifyDomainPage = withCardStateProvider(() => {
       .then(async res => {
         await resolve();
         if (res.verification?.status === 'verified') {
-          return navigate(`../../../domain/${res.id}?mode=select`);
```
[the remark this answers is not in the document]
Comment (desiprisg) [reply packages/clerk-js/src/ui/components/OrganizationProfile/VerifyDomainForm.tsx:62]: yes

File: packages/clerk-js/src/ui/components/OrganizationProfile/AddDomainForm.tsx (hunk 2 above)
Comment (desiprisg) [reply packages/clerk-js/src/ui/components/OrganizationProfile/AddDomainForm.tsx:41]: yes

File: packages/clerk-js/src/ui/components/OrganizationProfile/DomainList.tsx [hunk 5]
```diff
@@ -109,53 +105,43 @@ export const DomainList = withGate(
...
+                <Action.Card>
+                  <VerifiedDomainForm domainId={domain.id} />
+                </Action.Card>
+              </Action.Open>
+            </Action.Root>
           );
         })}
-        {(domains?.hasNextPage || domains?.isFetching) && (
+
+        {(domains?.hasNextPage || domains?.isFetching) && domains.data.length === 0 && (
```
Comment (panteliselef) [inline packages/clerk-js/src/ui/components/OrganizationProfile/DomainList.tsx:144]: Wouldn't this break the fetching of subsequent pages ?

File: packages/clerk-js/src/ui/elements/FormContent.tsx [hunk 6]
```diff
@@ -5,6 +5,11 @@ import { Col, descriptors } from '../customizables';
 import type { PropsOfComponent } from '../styledSystem';
 import { Card, Header, useCardState } from './index';
 
+export type FormProps = {
```
Comment (panteliselef) [inline packages/clerk-js/src/ui/elements/FormContent.tsx:8]: 🙃
 ```suggestion
export type CommonFormButtonProps = {
```

File: packages/localizations/src/en-US.ts [hunk 7]
```diff
@@ -604,14 +600,39 @@ export const enUS: LocalizationResource = {
...
+        subtitle:
+          'Allow users to join the organization automatically or request to join based on a verified email domain.',
+        primaryButton: 'Add domain',
+        menuAction__verify: 'Verify',
+        menuAction__remove: 'Delete',
+        menuAction__manage: 'Manage',
+        //TODO-RETHEME remove
+        unverifiedDomain_menuAction__verify: 'Verify domain',
+        //TODO-RETHEME remove
+        unverifiedDomain_menuAction__remove: 'Delete domain',
```
Comment (panteliselef) [inline packages/localizations/src/en-US.ts:626]: Is this for later ?

File: packages/clerk-js/src/ui/components/OrganizationProfile/DomainList.tsx (hunk 5 above)
Comment (desiprisg) [reply packages/clerk-js/src/ui/components/OrganizationProfile/DomainList.tsx:144]: why

**Pull request merged**